content/en/docs/concepts/services-networking/service.md

```diff
@@ -16,7 +16,7 @@ weight: 10
 
 <!-- overview -->
 
-{{< glossary_definition term_id="service" length="short" >}}
+{{< glossary_definition term_id="service" length="short" prepend="In Kubernetes, a Service is" >}}
```

Comment:
nit: do you need to clarify every sentence with `In Kubernetes, a Service is` instead of `A Kubernetes Service` or `A Service is`?

Reply:
I wanted to emphasize that I'm talking about what a Service is to Kubernetes, and that this isn't an explanation of the wider [service](https://en.wikipedia.org/wiki/Network_service) concept in IT.

Reply:
Okay, sure. The phrase `In Kubernetes` is used about seven times on the page,
https://deploy-preview-38054--kubernetes-io-main-staging.netlify.app/docs/concepts/services-networking/service/